tests/parsers/fixtures/pw/failed_dexx_negative/aiida.out

```diff
@@ -0,0 +1,152 @@
+
+     Program PWSCF v.6.4.1 starts on 14Oct2019 at  8:41:38 
+
+     This program is part of the open-source Quantum ESPRESSO suite
+     for quantum simulation of materials; please cite
+         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
+         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
+          URL http://www.quantum-espresso.org", 
+     in publications or presentations arising from this work. More details at
+     http://www.quantum-espresso.org/quote
+
+# NOTE: roughly 3870 lines of output removed.
+
+ %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+     Error in routine electrons (1):
+     dexx is negative!  Check that exxdiv_treatment is appropriate for the system, or ecutfock may be too low
+ %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+     stopping ...
```

Comment:
Can you just keep this block and remove all the others, since they are just duplicates

Reply:
Sure thing! Do you want to keep the standard QE header? Maybe just the line with the version number?

Reply:
Yes, let's keep the header but just remove the duplicated error message blocks

Reply:
Alright, done 👍